rfcs/convergence/make-styles-no-functions.md

```diff
@@ -0,0 +1,94 @@
+# Problem
+
+Currently `makeStyles()` implements two on how styles can be defined:
+
+```ts
+makeStyles({
+  rootA: { color: 'red' },
+  rootB: theme => ({ color: theme.tokenB }),
+});
+```
+
+`theme` is typed strictly and offers tokens from `@fluentui/react-theme`. This tokens shape is not extensible for customers.
+
+# Solution
+
+## Export tokens separately
+
+Tokens are just CSS variables, we should be clear with customers on what they are using. This also removes need in `useTheme()` hook as tokens can be accessed directly.
```

Comment:
```suggestion
Tokens are just CSS variables, we communicate this fact to customers so that they understand clearly what they are using. This also removes need in `useTheme()` hook as tokens can be accessed directly.
```